docs/source/usersguide/mgxs_library.rst

```diff
@@ -0,0 +1,290 @@
+
+.. _group_structure:
+
+``<group_structure>`` Element
+-----------------------------
+
+The ``<group_structure>`` element has no attributes and should be provided as a
+monotonically increasing list of bounding energies, in MeV, for a number of
+groups. To provide proper energy boundaries, the length of the data within the
+``<group_structure>`` element should be one more than the number of groups in
+the problem.  For example, a two-group problem could be specified as:
+
+.. code-block:: xml
+
+    <group_structure> 0.0 0.625E-6 20.0 </group_structure>
+
+*Default*: None, this must be provided.
+
+.. _inverse_velocities:
+
+``<inverse_velocities>`` Element
+--------------------------------
+
+The ``<inverse_velocities>`` element optionally indicates the average
+inverse velocity corresponding to each of the groups in the problem.
+This element should therefore be an array with a length which matches the
+number of groups set in the groups_ element.
+
+*Default*: Should this be needed by the presence of an ``inverse-velocity``
+score in the ``tallies.xml`` file and not provided in this element, OpenMC
+will simply convert the group mid-point energy to an inverse of the velocity
+and use this information for tallying.
+
+.. _xsdata:
+
+``<xsdata>`` Element
+--------------------
+
+The ``<xsdata>`` element contains the nuclide or material-specific meta-data as
+well as the actual cross section data. The following are the
+attributes/sub-elements required to describe the meta-data:
+
+  :name:
+    The name of the microscopic or macroscopic data set.
+
+    *Default*: None, this must be provided.
+
+  :alias:
+    An alternative name to use for the microscopic or macroscopic data set.
+
+    *Default*: If no alias is provided, it will adopt the value of ``name``.
+
+  :kT:
+    The temperature times Boltzmann's constant (in units of MeV) at which the
+    data was generated.
+
+    *Default*: Room temperature, 2.53E-8 MeV
+
+  :fissionable:
+    This element states whether or not the data in question is fissionable.
+    Accepted values are "true" or "false".
+
+    *Default*: None, this element must be provided.
+
+  :representation:
+    This element provides the method used to generate and represent the
+    multi-group cross sections.  That is, whether they were generated with
+    scalar flux weighting (or reduced to an equivalent representation)
+    and thus are angle-independent, or if the data was generated with angular
+    dependent fluxes and thus the data is angle-dependent.  The options are
+    either "isotropic" or "angle".
+
+    *Default*: "isotropic"
+
+  :num_azimuthal:
+    This element provides the number of equi-width bins that the azimuthal
+    angular domain is subdivided in the case of angle-dependent cross sections
+    (i.e., "angle" is passed to the ``representation`` element).
+
+    *Default*: If ``representation`` is "angle", this must be provided.  If
+    not, this parameter is not used.
```

Comment:
Add something to make it clear that these bins are equal angle rather than equal cosine